NOTICE

```diff
@@ -3,13 +3,13 @@ Copyright 2021 - present / Neuralmagic, Inc. All Rights Reserved.
 
 This product includes software developed at Neuralmagic, Inc. (https://www.neuralmagic.com).
 
-Source code in this repository is variously licensed under the Apache License
+Source code in the deepsparse GitHub repository is variously licensed under the Apache License
 Version 2.0, an Apache-compatible license.
 
 * For a copy of the Apache License Version 2.0, please see LICENSE
-  as included in this repository's top-level directory.
+  as included in the top-level directory of the deepsparse GitHub repository.
 
-* For a copy of the Neural Magic Engine License, please see LICENSE-NEURALMAGIC as included in this repository's top-level directory.
+* For a copy of the Neural Magic Engine License (which governs the DeepSparse Community Edition only), please see LICENSE-NEURALMAGIC as included in the top-level directory of the deepsparse GitHub repository.
```

Comment:
```suggestion
The Community Edition of the project's binary containing the DeepSparse Engine is licensed under the Neural Magic Engine License (https://github.com/neuralmagic/deepsparse/blob/main/LICENSE-NEURALMAGIC). Example files and scripts included in the deepsparse GitHub repository are licensed under the Apache License Version 2.0 (https://github.com/neuralmagic/deepsparse/blob/main/LICENSE) as noted.

The Enterprise Edition of the DeepSparse Engine (https://docs.neuralmagic.com/products/deepsparse-ent) requires a Trial License or can be fully licensed (https://neuralmagic.com/legal/master-software-license-and-service-agreement/) for production, commercial applications.
```

Reply:
I think this makes it clearer and covers both cases, no matter where you are looking at the Notice on GitHub, or downloaded in local env.

Reply:
@rgreenberg1 you should confirm the final version of this gets copied to the package as well.